package.json

```diff
@@ -48,10 +49,12 @@
     "text track"
   ],
   "scripts": {
-    "build": "browserify -s vttjs . -o dist/vtt.js",
+    "build": "npm run build-umd && npm run build-global",
```

Comment:
any reason not to run these in parallel with npm-run-all?

Reply:
There isn't other than that's another dep.